lib/process-services-cloud/src/lib/form/components/form-cloud.component.spec.ts

```diff
@@ -1056,6 +1057,27 @@ describe('FormCloudComponent', () => {
         });
     });
 
+
```

Comment:
double new line

Reply:
how is possible the linter didn't see it?

Reply:
Good question... anyway I removed that line

Reply:
created a task for it: https://alfresco.atlassian.net/browse/AAE-12747